vumi_http_retry/retries.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+
+def requests_key(prefix):
+    return ':'.join((prefix, 'requests'))
+
+
+def next_score(req):
+    dt = req['intervals'][req['attempts']]
+    return req['timestamp'] + dt
+
+
+def add_request(redis, prefix, req):
+    req = {
+        'timestamp': req['timestamp'],
+        'attempts': req.get('attempts', 0),
+        'request': req['request'],
+        'intervals': req['intervals']
+    }
```

Comment:
I'm +1 on storing the owner-id -- that way we at least know whose retry
request it is (without it if something goes wrong we won't be able to tell
easily).